doc/release-notes.md

```diff
@@ -31,15 +31,190 @@ Downgrading to a version older than v19.2.0 is not supported due to changes
+    - Introduces new format options `%h` and `%b`.
+        - `%b`: Replaced by the hash of the block including the transaction (set to `'unconfirmed'` if the transaction is not included).
+        - `%h`: Replaced by the block height (**-1** if not included).
+
+- **`-maxuploadtarget` Format Update**
+    - Now allows human-readable byte units `[k|K|m|M|g|G|t|T]`.
+        - **Example:** `-maxuploadtarget=500g`.
+        - **Constraints:** No whitespace, `+`, `-`, or fractions allowed.
+        - **Default:** `M` if no suffix is provided.
+
+## New Functionality
+
+- **`coinjoinsalt` RPC Command**
+    - Allows manipulation of a CoinJoin salt stored in a wallet.
+        - `coinjoinsalt get`: Fetches an existing salt.
+        - `coinjoinsalt set`: Allows setting a custom salt.
+        - `coinjoinsalt generate`: Sets a random hash as the new salt.
+
+- **LLMQ Type Enhancement**
+    - **LLMQ_50_60** is enabled for **Devnet** networks.
+    - Necessary for testing on a large Devnet.
+
+## P2P and Network Changes
+
+- **Improved Onion Connectivity**
+    - Nodes with onion connectivity will attempt to maintain at least **two outbound onion connections** and will protect these connections from eviction.
+    - **Benefit:** Ensures nodes capable of accessing the onion network maintain a few onion connections, allowing network messages to propagate even if non-onion IPv4 traffic is blocked.
+    - **Security Enhancement:** Enables P2P encryption for these peers.
+    - *(#6147)*
+
+- **DSQ Message Broadcast Update**
+    - Starting in protocol version **70234**, DSQ messages are broadcast using the inventory system instead of relaying to all connected peers.
+    - **Benefit:** Reduces bandwidth needs for all nodes, especially noticeable on highly connected masternodes.
+    - *(#6148)*
+
+- **Compressed Block Headers Request Limit**
+    - Starting in protocol version **70235**, The maximum number of compressed block headers that can be requested at once has been increased from **2000** to **8000**.
+    - **Applies to:** Compressed block headers only.
+    - **Benefit:** Potential for improved header sync performance.
+
+- **Multi-Network Connectivity Enhancement**
+    - Nodes with multiple reachable networks will actively try to maintain at least one outbound connection to each network.
+    - **Benefits:**
+        - Improves resistance to eclipse attacks.
+        - Enhances network-level resistance to partition attacks.
+    - **User Impact:** Users no longer need to perform active measures to ensure connections to multiple enabled networks.
+
+- **BIP324 Encrypted Communication (Experimental)**
+    - Dash Core now experimentally implements [BIP324](https://github.com/bitcoin/bips/blob/master/bip-0324.mediawiki), introducing encrypted communication for P2P network traffic.
+    - **Opt-In Adoption**
+      - **Enable Encryption:** Users can opt-in to use BIP324 by adding the `-v2transport=1` flag to their Dash Core configuration.
```

Comment:
It would only be `-v2...` if passing as cli arg
```suggestion
      - **Enable Encryption:** Users can opt-in to use BIP324 by adding the `v2transport=1` to their Dash Core configuration.
```